pkgs/development/python-modules/EasyOCR/default.nix

```
@@ -0,0 +1,61 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, hdf5
+, numpy
+, opencv3
+, pillow
+, pyaml
+, pyclipper
+, python-bidi
+, pytorch
```

```suggestion
, torch
```
pytorch is aliased to torch.